pkg/scheme/listers.go

```diff
@@ -31,6 +32,7 @@ func GenericListers() []client.ObjectList {
 		&corev1.SecretList{},
 		&corev1.ServiceAccountList{},
 		&corev1.ServiceList{},
+		&policyv1.PodDisruptionBudgetList{},
```

Comment:
suddenly a new lister appears? How come?

Reply:
Got warnings in one of the test suites (which should supposedly be an error, in previous versions)